tests/Dictionary_1.py

```diff
@@ -124,14 +124,14 @@ def testBadDictionary(self):
         p = Policy(self.getTestDictionary("conflict_occurs_policy_1.paf"))
         ve = ValidationError("Dictionary_1.py", 1, "testBadDictionary")
         d.validate(p, ve.cpp)
-        self.assert_(ve.getErrors("1to0") == ValidationError.TOO_MANY_VALUES)
-        self.assert_(ve.getErrors("2to1") == ValidationError.NOT_AN_ARRAY)
+        self.assert_(ve.getErrors("1to0") == int(ValidationError.TOO_MANY_VALUES))
+        self.assert_(ve.getErrors("2to1") == int(ValidationError.NOT_AN_ARRAY))
```

Comment:
I'm unsure if we want to make the `assert_` change here now. Changing unit tests while changing code (although sometimes required) is dangerous. But I could do it in a separate commit I suppose. I'll do that.